intel_extension_for_transformers/llm/runtime/graph/CMakeLists.txt

```diff
@@ -91,6 +91,7 @@ option(NE_GELU_VEC               "neural_engine: enable vec in gelu"
 if (NE_GELU_VEC)
     add_compile_definitions(NE_GELU_USE_VEC)
 endif()
+option(PYTHON_API              "neural_engine: use python api"                                  OFF)
```

Comment:
Maybe it should start with `NE_` as other options?

Reply:
thanks, update it here: https://github.com/intel/intel-extension-for-transformers/pull/413